src/auth/src/credentials/user_account.rs

```diff
@@ -188,26 +194,101 @@ impl Builder {
         self
     }
 
-    /// Returns a [Credentials] instance with the configured settings.
+    /// Configure the retry policy for fetching tokens.
     ///
-    /// # Errors
+    /// The retry policy controls how to handle retries, and sets limits on
+    /// the number of attempts or the total time spent retrying.
     ///
-    /// Returns a [CredentialsError] if the `authorized_user`
-    /// provided to [`Builder::new`] cannot be successfully deserialized into the
-    /// expected format. This typically happens if the JSON value is malformed or
-    /// missing required fields. For more information, on how to generate
-    /// `authorized_user` json, consult the relevant section in the
-    /// [application-default credentials] guide.
+    /// ```
+    /// # use google_cloud_auth::credentials::user_account::Builder;
+    /// # use gax::retry_policy;
+    /// # use gax::retry_policy::RetryPolicyExt;
```

Comment:
It may be hard to read the example if you don't see these:

```suggestion
    /// use gax::retry_policy::{self, RetryPolicyExt};
```

Reply:
I realized the samples are based on the existing client builder and request builder examples.  Maybe we should fix those too